src/components/Menu/config/config.ts

```diff
@@ -0,0 +1,109 @@
+import { MenuItemsType, DropdownMenuItemType, menuStatus } from '@pancakeswap/uikit'
+import { ContextApi } from 'contexts/Localization/types'
+
+export type ConfigMenuItemsType = MenuItemsType & { hideSubNav?: boolean }
+
+const config: (t: ContextApi['t']) => ConfigMenuItemsType[] = (t) => [
```

Comment:
Not a big thing but I wonder if this file should be `/config/index.ts`?

Reply:
There's two config files in this folder: `config` and `footerConfig`